source/MRMesh/MRObjectPointsHolder.h

```diff
@@ -83,21 +83,24 @@ class MRMESH_CLASS ObjectPointsHolder : public VisualObject
     /// returns the amount of memory this object occupies on heap
     [[nodiscard]] MRMESH_API virtual size_t heapBytes() const override;
 
-    /// sets maxRenderingPoints from rendering discretization, each \param val -th point will be displayed on screen
-    /// \detail convenient way to set maxRenderingPoints
-    /// maxRenderingPoints is set so: validPointsCount / val
-    MRMESH_API void setRenderDiscretization( int val );
-
-    /// returns rendering discretization, each N-th point will be displayed on screen
-    /// \detail used for rendering
+    /// returns rendering discretization
+    /// display each `renderDiscretization`-th point only (TODO: elaborate)
+    /// \detail defined by maximum rendered points number as:
+    /// \ref numValidPoints() / \ref getMaxRenderingPoints() (rounded up)
+    /// updated when setting `maxRenderingPoints` or changing the cloud (setting `DIRTY_FACE` flag)
     int getRenderDiscretization() const { return renderDiscretization_; }
 
+    /// default value for maximum rendered points number
+    static constexpr int MaxRenderingPointsDefault = 1'000'000;
+    /// recommended value for maximum rendered points number to disable discretization
+    static constexpr int MaxRenderingPointsUnlimited = std::numeric_limits<int>::max();
+
```

Comment:
why do we need it?

Reply:
Important use case is to disable discretization for some tools. Works better than `std::max( MaxRenderingPointsDefault, numValidPoints() )`.